docs/byoh-instance-pre-requisites.md

```diff
@@ -1,7 +1,6 @@
 # BYOH Instance Pre-requisites
 
 The following pre-requisites must be fulfilled in order to add a Windows BYOH node.
-* The Docker container runtime must be installed on the instance.
```

Comment:
The `containers feature` must be enabled in the Windows instance, is that documented somewhere?

Reply:
That is part of 2022 server. we should address that as part of 2022 server support.